scripts/generate

```diff
@@ -495,14 +521,17 @@ def ('tocListItem')
     ({})
     ([Heading, Html.Type])
     (function recur({level, id, html, subheads}) {
+       const pattern = '<a [^>]*>([^<]*)</a>';
        const a = el ('a')
                     ({href: '#' + id})
-                    (Just (Html (Html.unwrap (html)
-                                 .replace (/<a [^>]*>([^<]*)<[/]a>/g,
-                                           ($0, $1, idx) =>
-                                             when (K (idx === '<code>'.length))
-                                                  (wrap ('<b>') ('</b>'))
-                                                  ($1)))));
```

Comment:
This code actually uses the index provided by `String#replace`. The new code does not, and is clearer. :)